src/core.c/Distribution/Utils.pm6

```diff
@@ -0,0 +1,156 @@
+my role Distribution::Utils {
```

Comment:
Maybe this should just live in the same file as `CompUnit::Repository::Distribution`. That is the only spot it is used, and it is already a private class (so we shouldn't be concerned with the class name and path name matching).